specs.json

```diff
@@ -1229,7 +1229,16 @@
       "voiceMs": "number",
       "silenceMs": "number",
       "strategy": "string",
-      "mode": "number"
+      "mode": "number",
+      "vendor": {
+        "type": "string",
+        "enum": [
+          "native",
```

Comment:
actually lets change this from native to webrtc, because libfvad is basically the webrtc vad code lifted into its own library